code/deja/python/lab_06.py

```diff
@@ -0,0 +1,53 @@
+'''
+Lab 6: ARI 
+22 Mar 23
+'''
+import string
+file_name = "The Raven"
+file = open("the_raven.txt")
+contents = file.read()
+file.close
```

Comment:
Looks like you forgot the parenthesis at the end of file.close. This means you are not calling the function to close your file.